doc/fips.md

```diff
@@ -0,0 +1,132 @@
+# FIPS support
+
+libspdm 3.0.0 starts adding [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final) support for [Cryptographic Algorithm Validation Program (CAVP)](https://csrc.nist.gov/Projects/cryptographic-algorithm-validation-program). The FIPS boundary is libspdm library plus crypto library.
+
+NOTE: libspdm did not get any FIPS certificate.
```

Comment:
suggest rewording:
The SPDM WG does not plan to obtain FIPS 140-3 CMVP or CAVP certification for libspdm.

Reply:
done.